torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -680,8 +680,46 @@ def postprocessor(output):
                 # non-retraceable. To support retracing, we can add a repack node to the
                 # FX graph. For example:
                 # mkldnn._linear_pointwise <- repack_linear_wgt <- packed_wgt_for_template
+                wgt_tensor_users = 0
+                wgt_tensor = V.graph.constants[W_node.get_name()]
```

Comment:
nit: `W` that was initialized above is the same as this `wgt_tensor`. Don't need to create a new one. Also, for consistency, naming `wgt_tensor_users` as `W_tensor_users` is better?

Reply:
Thanks for the comment and changed the name.